docker/testdb/Dockerfile

```diff
@@ -1,3 +1,3 @@
-FROM camptocamp/postgres:9.6
+FROM camptocamp/postgres:12
```

Comment:
This is a little uncomfortable as we cannot import new administrative divisions anymore, nor locally nor on OpenShift.

Reply:
That's true, will keep it as is for now to stay consistent with current Openshift cluster